src/main/scala/com/evolutiongaming/conhub/transport/SendMsg.scala

```diff
@@ -79,6 +85,7 @@ object SendMsg {
 
     var state = Map.empty[Address, Channel]
 
+    val conhubRole = conf.toOption.flatMap(_.getOpt[String]("role")).getOrElse(DefaultRole)
```

Comment:
pass role as argument, and maybe default to the one from ActorSystem.settings.config.